lib/matplotlib/tests/test_inline_01.ipynb

```diff
@@ -0,0 +1,112 @@
W0aWXHokkBWGk1apckRLPQ7AJndVI2qw9dQKuBjyYF4OoofRshzKVZBU5mbgGvb0gHYO7IbgR3bqNyRUI0L80mcApLypm14hDFZQaGdm1D+IguapckRLPTLAJHURT+sj6d01cKcXOyI3qi9G2EUEOzCJxVP2WzLuUiVloNHz0RSJtWjW/NKyGagyYfOEcv5fPXf2cA8MLobgzs1FrlioRovpp04BQUlxEem0xpuYERvm15LqSz2iUJ0aw12cBRFIXI79I4e7UQD509CyYEoJW+jRCqarKB8/UPWWw6koO1VsMnkwNxaWmrdklCNHtNMnDSLuh5Z+NRAF55qDv9OrioXJEQAuoROAkJCTz88MN4enqi0WhYv379XffZu3cvffv2xc7Oji5duhATE1OPUusm/5e+TYWBkX5uzBjq02DPJYQwjtGBU1hYiL+/P4sXL67T+LNnzzJu3DhGjBhBamoqc+fOZcaMGWzfvt3oYu9GURReXn2ErOtFtHNuwYI/+KPRSN9GiMbC6LXFx4wZw5gxY+o8funSpfj4+LBgwQIA/Pz82LdvH9HR0YSGht5xn5KSEkpKSqpu5+fn1+m5Yg6cY1tGLjZWGhZP6YvOwabOdQohGl6D93ASExMZOXJktftCQ0NJTEyscZ+oqCh0Ol3V5u3tfdfnKS6r4PPvKxeve22sHwHezvdUtxDC9Bo8cHJzc3Fzq76YnJubG/n5+dy+ffuO+0RGRqLX66u27Ozsuz6PvY0V68MH81KoL38M7miK0oUQJmb0n1TmYGdnh52d8V8/aOtoJ1/KFKIRa/AZjru7O3l5edXuy8vLw8nJiRYtZO0nIZqTBg+coKAgdu3aVe2+uLg4goKCGvqphRCNjNGBc+vWLVJTU0lNTQUqP/ZOTU0lKysLqOy/TJ06tWr8c889x5kzZ3j55Zc5fvw4n376Kd9++y3z5s0zzREIISyG0T2cpKQkRowYUXV7/vz5AEybNo2YmBhycnKqwgfAx8eHzZs3M2/ePBYtWoSXlxeff/55jR+J34miKEDdPx4XQpjPL+/LX96ntdEodRmlsgsXLtTpo3EhhHqys7Px8vKqdYxFBI7BYODSpUs4OjrWeuZwfn4+3t7eZGdn4+TkZMYKG44ck2VozsekKAoFBQV4enqi1dbepWmUH4v/N61We9fk/E9OTk5N5kX/hRyTZWiux6TT6er0WE3y2+JCiMZJAkcIYTZNKnDs7Ox444036nWWcmMlx2QZ5JjqxiKaxkKIpqFJzXCEEI2bBI4QwmwkcIQQZiOBI4QwGwkcIYTZWFzgLF68mI4dO2Jvb8/AgQP58ccfax2/evVqunfvjr29Pb1792bLli1mqrTujDmmmJgYNBpNtc3e3t6M1d5dY1/Zw1jGHs/evXt/8xppNBpyc3PNU3AdREVF0b9/fxwdHXF1dSUsLIzMzMy77nev7yeLCpxVq1Yxf/583njjDZKTk/H39yc0NJTLly/fcfyBAweYNGkS06dPJyUlhbCwMMLCwkhPTzdz5TUz9pig8lTznJycqu38+fNmrPjuGvPKHvVh7PH8IjMzs9rr5Orq2kAVGi8+Pp7w8HAOHjxIXFwcZWVljB49msLCwhr3Mcn7SbEgAwYMUMLDw6tuV1RUKJ6enkpUVNQdx0+YMEEZN25ctfsGDhyoPPvssw1apzGMPaYvv/xS0el0Zqru3gHKunXrah3z8ssvKz179qx238SJE5XQ0NAGrKx+6nI8e/bsUQDlxo0bZqnJFC5fvqwASnx8fI1jTPF+spgZTmlpKYcOHaq2AoRWq2XkyJE1rgBRnxUjzKk+xwSVF0Hr0KED3t7ejB8/noyMDHOU22Aa++tUXwEBAXh4eDBq1Cj279+vdjm10uv1ALi41LxKrSleJ4sJnKtXr1JRUXHHFSBq+tu4phUjGsvf0vU5Jl9fX7744gs2bNjA119/jcFgIDg4mAsXLpij5AZRn5U9GjMPDw+WLl3K2rVrWbt2Ld7e3gwfPpzk5GS1S7sjg8HA3LlzGTx4ML169apxnCneTxZxeQrxq6CgoGrXgw4ODsbPz4/PPvuMd955R8XKxC98fX3x9fWtuh0cHMzp06eJjo5m+fLlKlZ2Z+Hh4aSnp7Nv374Gfy6LmeG0adMGKyurO64A4e7ufsd9aloxoqbx5lafY/pvNjY2BAYGcurUqYYo0Syaw8oeAwYMaJSvUUREBJs2bWLPnj13veaUKd5PFhM4tra29OvXr9oKEAaDgV27dtW4AkRjXzGiPsf03yoqKkhLS8PDw6Ohymxwjf11MoXU1NRG9RopikJERATr1q1j9+7d+Pj43HUfk7xO9e1qq2HlypWKnZ2dEhMToxw9elR55plnFGdnZyU3N1dRFEV56qmnlFdffbVq/P79+xVra2vlgw8+UI4dO6a88cYbio2NjZKWlqbWIfyGscf01ltvKdu3b1dOnz6tHDp0SHniiScUe3t7JSMjQ61D+I2CggIlJSVFSUlJUQBl4cKFSkpKinL+/HlFURTl1VdfVZ566qmq8WfOnFEcHByUl156STl27JiyePFixcrKStm2bZtah1CNsccTHR2trF+/Xjl58qSSlpamzJkzR9FqtcrOnTvVOoTfmDlzpqLT6ZS9e/cqOTk5VVtRUVHVmIZ4P1lU4CiKonz88cdK+/btFVtbW2XAgAHKwYMHq/5t2LBhyrRp06qN//bbb5Vu3boptra2Ss+ePZXNmzebueK7M+aY5s6dWzXWzc1NGTt2rJKcnKxC1TX75WPh/95+OY5p06Ypw4YN+80+AQEBiq2trdKpUyflyy+/NHvdNTH2eP7+978rnTt3Vuzt7RUXFxdl+PDhyu7du9UpvgZ3Oh6g2v97Q7yf5Ho4QgizsZgejhDC8kngCCHMRgJHCGE2EjhCCLORwBFCmI0EjhDCbCRwhBBmI4EjhDAbCRwhhNlI4AghzEYCRwhhNv8PCRnDMpOE+9MAAAAASUVORK5CYII=",
```

Comment:
Do we need to commit this whole output?

Reply:
No, I can clean the notebook to remove the outputs. I kept it in to be consistent with `test_nbagg_01.ipynb`, which is what this is based on. But neither actually compare the output image pixels.

As an aside it would be great to have some image-comparison testing of some combination of Matplotlib and IPython/Jupyter, but I cannot find a good place to put them. There is an excellent image-based testing framework  used in Jupyter projects but it would bring in some complicated test dependencies so it is not appropriate here.

Reply:
Done

Reply:
It would be best if this were removed on the initial commit instead of a later one, unless you intend for this entire PR to be squashed.

Reply:
I'll squash.